react-backend/routes/postActivity.js

```diff
@@ -23,8 +23,7 @@ router.post('/', (req, res, next) => {
         putVisitsData(visitToAdd.hash, visitToAdd.activity, (error, result) => {
           if (error) {
             console.log('I am an error from putVisitsData ', error);
-          } else {
-            res.send();
+            res.status(500).send(err);
           }
```

Comment:
We still need to send something back to the front end in the case of a success, even if it's just a message saying "ok" or something similarly trivial.

Reply:
Sending "success" message now :)